src/olympia/blocklist/tasks.py

```diff
@@ -90,6 +90,8 @@ def import_block_from_blocklist(record):
                 'Kinto %s: Unable to break down regex into list; '
                 'attempting to create Blocks for guids matching [%s]',
                 kinto_id, guid_regexp)
+            # mysql doesn't support /d - only [:digit:]
```

Comment:
`\` not `/`

Reply:
doesn't support `\d` and doesn't support `/d` either! :)

Reply:
https://github.com/mozilla/addons-server/pull/13912/commits/e67bdbddcdd0933eb11edf9bcafa7d6e17dd3c62